asminc/apple2.inc

```diff
@@ -85,3 +85,6 @@ IOUDISOFF   :=  $C07F   ; Enable IOU
 
 ; Control Your Apple
 CYAREG  :=      $C036   ; Bits 0-3=disk detect 4=shadow all banks 7=fast
+
+; Max filename length
+FILENAME_MAX := 64
```

Comment:
* I think FILENAME_MAX doesn't belong here. This file is about Apple II hardware attributes, not about Apple II software attributes. It fits much better in libsrc/apple2/mli.inc.
* The definition should use `=`, not `:=` as it doesn't denote an address.
* Given that the C definition is `64+1` I'm wondering if it might be preferrable to use `64+1` here too.